app/views/shared/_supporting_information.html.erb

```diff
@@ -14,11 +14,14 @@
     <div class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
       <ul class="govuk-list govuk-list--bullet">
         <% if @planning_application.constraints %>
```

Comment:
this should be `@planning_application.constraints.present?` if we want this check, as the default is an empty array, which is "thruthy" - but also it doesn't seem to be doing anything anyway, so we can remove it.

Reply:
Oh yes, we can remove this now. Before we had a default, we needed it because it would have thrown an error if there weren't any.

Reply:
Removed.